README.md

```diff
@@ -46,41 +46,26 @@ This package contains all the custom [actions](http://wiki.ros.org/actionlib/Tut
 * pcl_ransac_ros  
 This contains the main ROS package. All your package-specific code goes here.  
     * cfg  
-        * template.cfg  
```

Comment:
Maybe let this part remain? It has references for writing dynamic reconfigure files

Reply:
we can add that later, when we add dyncfg to the template